test/messages/PutMessageRequestTest.cpp

```diff
@@ -0,0 +1,36 @@
+#include "gtest/gtest.h"
+#include "messages/PutMessageRequest.h"
+#include "messages/Constants.h"
+#include <string>
+
+TEST(PutMessageRequest, VerifySerialize)
```

Comment:
Just noticed that this test is incomplete. I think I will start creating more helper functions to easily extract values and offsets from messages.